pkg/tests/apis/types_discoverty.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
I think this filename should be `types_discovery.go`

Reply:
omg -- what is the "t" for 🤔